soc/arm/musca/Kconfig.soc

```diff
@@ -4,20 +4,14 @@
 config SOC_SERIES_MUSCA
 	bool
 	select SOC_FAMILY_ARM
-	help
-	   ARM v2m MUSCA MCU Series
 
 config SOC_V2M_MUSCA_B1
 	bool
 	select SOC_SERIES_MUSCA
-	help
-	  ARM Cortex-M33 SMM-SSE-200 on V2M-MUSCA-B1
```

Comment:
The description provided here looks pretty useful to me -- just by looking at `SOC_V2M_MUSCA_B1`, it is not very easy to figure out that this is for SSE-200 reference system.